content/en/docs/installation/armory-agent-quick.md

```diff
@@ -0,0 +1,166 @@
+---
```

Comment:
I'd recommend moving the install docs for Agent to the admin section. Currently, our install docs focus around installing just Armory/Spinnaker and not enabling any of the bells and whistles. 

The bells and whistles like Terraform Integration and Pipelines as Code are in the Admin section.

This would also allow you to move the intro material on this page to the top-level page for the agent, which is where I'd logically expect it.

Reply:
It was top level in the draft, I moved it to Installation to de-emphasize. I am okay being moved to `Administration` but that section is a mess. I guess we could show the way. I'll let @aimeeu decide.

Reply:
I’d like to see all the agent docs in a single, top-level folder outside of Administration. My reasoning:  The Armory Agent for Kubernetes is a standalone product that's deployed on Spinnaker or the Armory platform. All the content in a top-level folder makes it easy for readers to find the product, read why they should use it, and see how to install it.   The Armory Agent differs from Terraform Integration and PaC in that you don't need the Armory platform to use the Armory Agent.

Reply:
I don't think it is a standalone product as in it does nothing without a Spinnaker instance. I prefer it under `Administration` or split like it is.

Reply:
My point is that a user doesn't need an **Armory** installation to deploy the agent. The content under Admin is supposed to be specific to an on-prem Armory installation. If there is still content in Admin that is for open source Spinnaker, it should be moved to spinnaker.io. 
If I overhear somebody talking about Armory's new service for Spinnaker and I go to our docs site, I have to search for the new <insert name here> service. I don't know to look under Administration, which btw will change to something like Admin (Self-Hosted) to differentiate the content from Admin (SaaS).